tests/simulator/api.js

```diff
@@ -76,11 +76,13 @@ const shellyMiddleware = (req, res, next) => {
   if (req.originalUrl === "/shelly") {
     res.end(JSON.stringify({ gen: 2 }));
   } else if (req.originalUrl === "/rpc/Shelly.ListMethods") {
-    res.end(JSON.stringify({ methods: ["Switch.GetStatus"] }));
+    res.end(JSON.stringify({ methods: ["Switch.GetStatus","Temperature.GetStatus"] }));
```

Comment:
Muss das überhaupt geändert werden? Aktuell wird der Shelly Mock nur in einem Test verwendet wo über die UI ein Shelly Schalter angelegt wird, und wir testen dass sich evcc korrekt verhält wenn das Gerät nach Neustart nicht mehr verfügbar (mock/simulator beendet) ist.

Reply:
Das Thema Temperatur und die Werte werden nicht durch den E2E Test geprüft.

Reply:
ok, danke, dann reverte ich das wieder :-)